src/main/java/codeu/controller/ConversationServlet.java

```diff
@@ -28,6 +29,8 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import static codeu.model.data.Conversation.*;
```

Comment:
Static imports generally go above the regular imports. And try to avoid using `*` unless there are a large number of members that need to be imported